feature-libs/my-account/organization/components/unit/units.config.ts

```diff
@@ -192,94 +188,81 @@ export function unitsTableConfigFactory(): TableConfig {
 
 export const unitsTableConfig: TableConfig = {
   table: {
-    [OrganizationTableType.UNIT_USERS]: [
-      {
-        headers: [{ key: 'summary' }, { key: 'link' }],
-        hideHeader: true,
+    [OrganizationTableType.UNIT_USERS]: {
+      cells: ['name', 'roles'],
+      options: {
```

Comment:
units/${code}/users - missing roles headers in table
```suggestion
      options: {
        hideHeader: false,
```

Reply:
I'm holding off with changing this once more in this branch. We've received a new UX for the roles changes, which looks great, but I like to align with UX team once more.